### PR TITLE
updated ru.yml

### PR DESCRIPTION
**Pull request by agm1988**
added missing translations to ru.yml

### CODE REVIEW

File: locales/ru.yml [hunk 1]
```diff
@@ -54,3 +82,18 @@ ru:
     unlocks:
       new:
         resend_unlock_instructions: "Выслать подтверждение заново"
+      send_instructions: В течении нескольких минут Вам на почту будет выслана инструкция по разблокировке аккаунта
+      send_paranoid_instructions: Если Ваш аккаунт существует, в течении нескольких минут Вам на почту будет выслана инструкция по разблокировке аккаунта
+      unlocked: Ваш аккаунт успешно разблокирован. Войдите чтобы  продолжить
```
Comment (denispeplin) [inline locales/ru.yml]: Extra space.

File: locales/ru.yml [hunk 2]
```diff
@@ -29,8 +42,13 @@ ru:
       edit:
         change_your_password: "Изменить пароль"
         new_password: "Новый пароль"
-        confirm_new_password: Повторите новый пароль"
+        confirm_new_password: Повторите новый пароль
         change_my_password: "Изменить мой пароль"
+      no_token: Данная страница недоступна пока вы не перейдете по ссылке из письма по сбросу пароля. Если вы перешли по ссылке из письма, убедитесь что скопировали ссылку полностью
+      send_instructions: В течении нескольких минут к Вам на почту прийдет письмо с инструкцией по сбросу пароля
```
Comment (denispeplin) [inline locales/ru.yml]: `прийдет` -> `придет`

File: locales/ru.yml [hunk 3]
```diff
@@ -39,8 +57,18 @@ ru:
         title: "Редактировать \"%{resource}\""
         we_need_your_current_password_to_confirm_your_changes: "введите текущий пароль для подтверждения изменений"
       new:
-        sign_up: "Регистрация"
+        sign_up: Регистрация
```
Comment (denispeplin) [inline locales/ru.yml]: What was changed except of quotes?

File: locales/ru.yml [hunk 4]
```diff
@@ -39,8 +57,18 @@ ru:
...
+      destroyed: Ваш аккаунт успешно удален.
+      signed_up: Добро пожаловать! Вы зарегистрировались успешно
+      signed_up_but_inactive: Вход выполнен успешно. Но Ваш аккаунт еще не активирован
+      signed_up_but_locked: Вход выполнен успешно, но ваш аккаунт заблокирован
```
Comment (denispeplin) [inline locales/ru.yml:64]: Some inconsistency with the line above. Dot there, comma here.

**Comment by denispeplin** [comment]
@hisea, we are using these translations in our application, I think that overall quality is good.
Two questions are left:
1. Are quotes mandatory?
2. Should be `squash` performed for this pull request?

**Comment by hisea** [comment]
@denispeplin Thanks for your PR.
1. Quotes are not mandatory as long as they are tested locally.
2. please `squash`, will merge after you `squash` the commits.

**Comment by denispeplin** [comment]
It's not mine, it's @agm1988's.
Ok, I'll ask him about squash.

**Comment by agm1988** [comment]
@hisea, I squashed commits, please check

**Comment by hisea** [comment]
Thank you @agm1988

**Comment by agm1988** [comment]
@hisea please update gem version

**Comment by hisea** [comment]
@agm1988 Updated gem version

**Comment by agm1988** [comment]
@hisea thanks!

**Pull request merged**